c2corg_ui/static/js/auth/authservice.js

```diff
@@ -89,32 +89,67 @@ app.Authentication.prototype.isAuthenticated = function() {
 /**
  * Checks if the current user has rights to access/edit the document.
  * TODO: rights to personal documents only for current user.
+ * @param {Array<string>} users
+ * @param {?string} imageType
  * @return {boolean}
  * @export
  */
-app.Authentication.prototype.hasEditRights = function(users) {
+app.Authentication.prototype.hasEditRights = function(users, imageType) {
```

Comment:
I don't find very elegant/convenient to pass an argument that is only relevant for a given type of document.
What about for instance passing a generic ``options`` argument where you could pass the image type when needed?